go/libraries/doltcore/sqle/sqlddl_test.go

```diff
@@ -1091,6 +1103,30 @@ func TestParseCreateTableStatement(t *testing.T) {
 	}
 }
 
+func newTestEngine(ctx context.Context, dEnv *env.DoltEnv) (*gms.Engine, *sql.Context) {
```

Comment:
This must be defined somewhere else right?

Reply:
kind of, the other one does the reverse, constructing an engine from a database. This is an empty dolt directory that needs a new DB. I probably could put more effort into figuring out how to manually create a dolt database and join it with the env, seemed like an equal amount of extra work